src/fu-util.c

```diff
@@ -2740,6 +2741,9 @@ main (int argc, char *argv[])
 		{ "no-metadata-check", '\0', 0, G_OPTION_ARG_NONE, &priv->no_metadata_check,
 			/* TRANSLATORS: command line option */
 			_("Do not check for old metadata"), NULL },
+		{ "no-remote-check", '\0', 0, G_OPTION_ARG_NONE, &priv->no_remote_check,
+			/* TRANSLATORS: command line option */
+			_("Do not check for download remotes to be enabled"), NULL },
```

Comment:
Maybe `Do not check if download remotes should be enabled`?

Reply:
yep that is better, adjusted